acceptance_tests/features/steps/view_collection_exercise_details_ready_for_review_state.py

```diff
@@ -42,17 +40,11 @@ def navigate_to_rsi_details(_):
 @then('the status of the 201811 collection exercise is Ready for Review')
 def rsi_201811_is_ready_for_review(_):
     collection_exercises = collection_exercise.get_collection_exercises()
-    ce_state = next((ce['exercise_ref'] for ce in collection_exercises if ce['exercise_ref'] == '201811'))
     # Status updated async so wait until updated
-    for i in range(5):
-        if collection_exercise.is_ready_for_review(ce_state):
-            break
-        time.sleep(1)
+    if not browser.is_text_present('Ready for review', wait_time=2):
```

Comment:
Not sure if this will work because the page won't be updated asynchronously although the data is. It is likely it will always wait for 2 seconds rather that up to 2 seconds